ddl/ddl_workerpool.go

```diff
@@ -88,38 +88,38 @@ func (wp *workerPool) tp() jobType {
 	return wp.t
 }
 
-// backfilWorkerPool is used to new backfill worker.
-type backfilWorkerPool struct {
+// backfillCtxPool is used to new backfill context.
```

Comment:
Should workpool things be maintenance and implement by dist framework?

Reply:
Do you recommend putting a new file directly or in a new directory?